js/view.js

```diff
@@ -1,114 +1,114 @@
-// View
-
 var View = (function() {
-  View = {};
+  return {
```

Comment:
Why this change? Too much repetition of `View`? I kept it the other way because I find the `class.functionName =` syntax way easier to read. I like to be able to scan the file and say "ah yes, `View` has a `setBuildListeners` method on it..."

It's a personal preference of mine. I find the object literal notation harder to read with the semicolons.